<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QDialog" name="RegistrationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>70</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>70</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelPasswordRecovery">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1001</width>
     <height>78</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;РЕГИСТРАЦИЯ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBackToAuth">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>АВТОРИЗАЦИЯ</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>571</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditMail">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QLineEdit" name="lineEditLogin">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPassword">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPasswordCheck">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelMail">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Почта</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLogin">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Логин</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPassword">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPasswordCheck">
            <property name="font">
             <font>
              <family>Gilroy</family>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="font">
         <font>
          <family>Gilroy</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ДАЛЕЕ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCreateAccount">
        <property name="font">
         <font>
          <family>Gilroy</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>СОЗДАТЬ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelCode">
        <property name="font">
         <font>
          <family>Gilroy</family>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Код</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCode">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Gilroy</family>
          <pointsize>22</pointsize>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
